### PR TITLE
OSDOCS#10758: Azure reorg

### PR DESCRIPTION
**Pull request by snarayan-redhat**
Version(s): 4.17

Issue: https://issues.redhat.com/browse/OSDOCS-10758

Link to docs preview:  https://76797--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_azure/preparing-to-install-on-azure

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

File: installing/installing_azure/ipi/installing-azure-private.adoc [hunk 1]
```diff
@@ -24,6 +24,13 @@ include::modules/installation-azure-user-defined-routing.adoc[leveloffset=+2]
 
 include::modules/installation-about-custom-azure-vnet.adoc[leveloffset=+1]
 
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../../networking/openshift_sdn/about-openshift-sdn.adoc#about-openshift-sdn[About the OpenShift SDN network plugin]
+
+* xref:../../../installing/install_config/configuring-firewall.adoc#configuring-firewall[Configuring your firewall]
+
```
Comment (bscott-rh) [inline installing/installing_azure/ipi/installing-azure-private.adoc:33]: This is interesting, did these lines get added somehow?

File: installing/installing_azure/ipi/installing-restricted-networks-azure-installer-provisioned.adoc [hunk 2]
```diff
@@ -28,15 +28,22 @@ Because the installation media is on the mirror host, you can use that computer
...
 
 include::modules/installation-about-custom-azure-vnet.adoc[leveloffset=+1]
 
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../../networking/openshift_sdn/about-openshift-sdn.adoc#about-openshift-sdn[About the OpenShift SDN network plugin]
+
+* xref:../../../installing/install_config/configuring-firewall.adoc#configuring-firewall[Configuring your firewall]
+
```
Comment (bscott-rh) [inline installing/installing_azure/ipi/installing-restricted-networks-azure-installer-provisioned.adoc:46]: same question here.

**Review by bscott-rh — COMMENTED** [review]
LGTM! left two questions that are probably just quirks of git. Nice work

File: installing/installing_azure/ipi/installing-azure-private.adoc (hunk 1 above)
Comment (snarayan-redhat) [reply installing/installing_azure/ipi/installing-azure-private.adoc:33]: In both places, I added them to address the xrefs present in module `modules/installation-about-custom-azure-vnet.adoc`.

Comment (bscott-rh) [reply installing/installing_azure/ipi/installing-azure-private.adoc:33]: Great, thank you

**Pull request merged**